src/frontends/pytorch/src/op/split.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/split.hpp"
+
+#include <climits>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+std::shared_ptr<ov::Node> translate_chunk_common(const NodeContext& context, const bool split_by_chunks) {
+    num_inputs_check(context, 2, 3);
+    auto num_chunks = context.const_input<int>(1);
+    auto dim = context.get_input(2);
+
+    std::shared_ptr<ov::Node> chunk;
+    if (split_by_chunks) {
+        chunk = context.mark_node(std::make_shared<v1::Split>(context.get_input(0), dim, num_chunks));
```

Comment:
Chunk cant be replaced by Split, because input can be not evenly divisible in that case VariadicSplit needs to be used, we have transformation for chunk, please remove it from ts at least

Reply:
Removed from TorchScript